applications/StructuralMechanicsApplication/custom_constitutive/elastic_isotropic_3d.cpp

```diff
@@ -69,17 +69,27 @@ namespace Kratos
         const double& NU    = MaterialProperties[POISSON_RATIO];
 
         //NOTE: SINCE THE ELEMENT IS IN SMALL STRAINS WE CAN USE ANY STRAIN MEASURE. HERE EMPLOYING THE CAUCHY_GREEN
-        if(Options.IsNot( ConstitutiveLaw::COMPUTE_STRAIN )) //large strains
+        if(Options.Is( ConstitutiveLaw::COMPUTE_STRAIN ))
```

Comment:
The way I did is the way it is supposed to be. You need to compute the strain when you set the flag `COMPUTE_STRAIN`